hydra-tui/src/Hydra/TUI/Drawing.hs

```diff
@@ -99,7 +105,15 @@ drawFocusPanelFinal :: NetworkId -> VerificationKey PaymentKey -> UTxO -> Widget
 drawFocusPanelFinal networkId vk utxo =
   padLeftRight 1 $
     txt ("Distributed UTXO, total: " <> renderValue (balance @Tx utxo))
-      <=> padLeft (Pad 2) (drawUTxO (\a -> withAttr (if a == mkVkAddress networkId vk then own else mempty) $ drawAddress a) utxo)
+      <=> padLeft
+        (Pad 2)
+        ( drawUTxO
+            (\a -> withAttr (if a == mkVkAddress networkId vk then own else mempty) $ drawAddress a)
```

Comment:
Suggestion: Remove duplication here? This seems to be the same function than line 92